schema/spacewalk/upgrade/susemanager-schema-4.4.6-to-susemanager-schema-4.4.7/100-update-recurring-action.sql

```diff
@@ -18,40 +18,49 @@ CREATE TABLE IF NOT EXISTS suseInternalState
   label             VARCHAR(128) NOT NULL
 );
 
-DELETE FROM suseInternalState;
-
 INSERT INTO suseInternalState (id, name, label)
-         VALUES (1, 'certs', 'Certificates');
```

Comment:
if the ID are the same I think you don't need a sub-select, you just need to specify `on conflict do nothing`

Reply:
ah, sorry, I probably didn't explain myself well :)